GitHubData/ExtensionMethods.cs

```diff
@@ -0,0 +1,38 @@
+// <copyright file="ExtensionMethods.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+using System.Collections.ObjectModel;
+
+namespace GitHubData;
+
+/// <summary>
+/// Provides helper methods that can be used throughout the library.
+/// </summary>
+public static class ExtensionMethods
+{
+    // TODO: Unit test this
+
+    /// <summary>
+    /// Converts the given list of (<typeparamref name="TKey"/>, <typeparamref name="TValue"/>) tuples
+    /// to a list of type <see cref="KeyValuePair{TKey,TValue}"/>.
```

Comment:
Refactor code docs to below:

```suggestion
    /// to a list of <see cref="IEnumerable{T}"/> items of type <see cref="KeyValuePair{TKey,TValue}"/>.
```